classes/page_group.php

```diff
@@ -61,6 +61,12 @@ protected function set_fuzzydurationcounts(array $data) {
      */
     public static function record_fuzzy_counts(profile $profile) {
 
+        // Do this only if both auto profiling and fuzzy counting is set.
+        if (!get_config('tool_excimer', 'enable_auto') ||
+            !get_config('tool_excimer', 'enable_fuzzy_count')) {
```

Comment:
This seems fine. I'm wondering if there are situations where you would only want to put the flag for enable_fuzzy_count here instead, e.g. you might want this to compute the fuzzy count for profiles where it isn't automatically profiled. An example might be a manually triggered cron run, that might have a flag set to profile that particular run.

What do you think?

Reply:
This was explicitly requested.